src/backend/distributed/deparser/deparse_text_search.c

```diff
@@ -68,6 +113,13 @@ AppendDefElemList(StringInfo buf, List *defelems)
 		}
 		first = false;
 
+		/* omit the value when the arg is null */
```

Comment:
could you explain when this could be null?
The current comment is pretty much telling what the if-statement is doing, not why.

Reply:
expanded the comments here.

```
		/*
		 * There are some operations that can omit the argument. In that case, we only use
		 * the defname.
		 *
		 * For example, omitting [ = value ] in the next query results in resetting the
		 * option to defaults:
		 * ALTER TEXT SEARCH DICTIONARY name ( option [ = value ] );
		 */
```